agent/backend/otel/policy.go

```diff
@@ -175,3 +184,37 @@ func (o *openTelemetryBackend) ValidatePolicy(otelConfig openTelemetryConfig) er
 
 	return nil
 }
+
+func SanitizePolicyData(policyData policies.PolicyData) (*policies.PolicyData, error) {
+	originalJSON, err := json.Marshal(policyData)
+	if err != nil {
+		return nil, err
+	}
+	policyDataClone := policies.PolicyData{}
```

Comment:
just out for standards, why not use the var here?

Reply:
Good point